test-cases/tests/upgrade/n07-verify-cve-upgrade.md

```diff
@@ -0,0 +1,66 @@
+---
+tags:
+  - happy-path
+---
+
+# N06 - Verify CVE upgrades override pending install plans
```

Comment:
```suggestion
# N08 - Verify CVE upgrades override pending install plans
```
Also - the filename should reflect the heading name (note the ID (N08) and the rest of the heading)